### PR TITLE
refactor(textarea): convert Light DOM LESS to SCSS

### PR DESCRIPTION
**Pull request by BY00565233**
JIRA: SURF-1901

### LGTM's
* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

File: src/scss/components/textarea/_index.scss [hunk 1]
```diff
@@ -1,14 +1,15 @@
-@import (reference) "mixins";
-@import (reference) "components/form/index";
+@import "mixins";
+@import "components/form/config";
+
+/// ----- LAYOUT ----------
+/// (columnar-flexbox)
```
Comment (100stacks) [inline src/scss/components/textarea/_index.scss]: Please fix typo:

```css
/// (flexbox - column layout)
```

File: src/scss/components/textarea/_index.scss [hunk 2]
```diff
@@ -42,41 +43,42 @@ hx-textarea-control {
...
   > textarea {
-    .TextControl(pristine);
+    @include hxTextControl;
 
     &:focus {
-      .TextControl(pristine-focus);
+      @include hxTextControl($focused: true);
     }
   }
 }//hx-textarea-control
```
Comment (100stacks) [inline src/scss/components/textarea/_index.scss]: Please remove comment.

File: src/scss/components/textarea/_index.scss (hunk 1 above)
Comment (BY00565233) [reply src/scss/components/textarea/_index.scss]: Changes have been done.

File: src/scss/components/textarea/_index.scss (hunk 2 above)
Comment (BY00565233) [reply src/scss/components/textarea/_index.scss]: Changes have been done.

File: src/scss/components/textarea/_index.scss (hunk 1 above)
Comment (CITguy) [reply src/scss/components/textarea/_index.scss]: wasn't a typo, but no worries

**Pull request merged**